scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -453,3 +453,98 @@ task PostprocessGermlineCNVCalls {
         File genotyped_segments_vcf = genotyped_segments_vcf_filename
     }
 }
+
+task CollectSampleQualityMetrics {
+    Array[File] genotyped_segments_vcf
+    Array[String] entity_ids
+
+    Int? maximum_number_events = 120
+
+    # Runtime parameters
+    String gatk_docker
+    Int? mem_gb
+    Int? disk_space_gb
+    Boolean use_ssd = false
+    Int? cpu
+    Int? preemptible_attempts
+
+    Int machine_mem_mb = select_first([mem_gb, 1]) * 1000
+
+    String dollar = "$" #WDL workaround for using array[@], see https://github.com/broadinstitute/cromwell/issues/1819
+
+    command <<<
+        set -e 
+
+        genotyped_segments_vcfs_array=(${sep=" " genotyped_segments_vcf})
+        entity_ids=(${sep=" " entity_ids})
+        for index in ${dollar}{!genotyped_segments_vcfs_array[@]}; do
+            NUM_SEGMENTS=$(grep -v '@' ${dollar}{genotyped_segments_vcfs_array[$index]} | wc -l)
+            if [ $NUM_SEGMENTS -lt ${maximum_number_events} ]; then
```

Comment:
How are we defining "event"? If that includes copy-neutral segments then this script is fine, but when I hear "event" I think of DELs/DUPs.

Reply:
It's number of non comment lines in the genotyped segments VCF file, so everything including copy-neutral segments (so there are at least 24 events in each sample)

Reply:
I think this could be a point of confusion for users. Can you change `maximum_number_of_events` to `maximum_number_of_segments` and clarify how it is defined (as you have here) in the comment at the workflow input?